src/Npgsql.NodaTime/NpgsqlNodaTimeExtensions.cs

```diff
@@ -61,7 +62,7 @@ public static INpgsqlTypeMapper UseNodaTime(this INpgsqlTypeMapper mapper)
                 {
                     PgTypeName = "interval",
                     NpgsqlDbType = NpgsqlDbType.Interval,
-                    ClrTypes = new[] { typeof(Period) },
+                    ClrTypes = new[] { typeof(Period), typeof(TimeSpan) },
```

Comment:
Forgot about `NpgsqlTimeSpan`.

Reply:
As a general rule, it's fine to leave out all the Npgsql date/time types (e.g. NpgsqlDate) - we're considering obsoleting/removing them anyway. Or you can delegate them as well, up to you (I doubt anyone actually needs them, especially when using the Nodatime plugin).